src/main/js/components/CnesReportApp.js

```diff
@@ -60,6 +61,13 @@ export default class CnesReportApp extends React.PureComponent {
     }
 
     componentDidMount() {
+        // Initialize compatibility check
+        isCompatible().then(isSupported => {
+            this.setState({ isSupported });
+            console.log(isSupported);

```

Comment:
Il est pertinent ce console.log ?
Autant pour tester pendant que tu fais ta branche, ça marche, autant après coup voir un "True" ou "False" dans les logs sorti de nul part je pense par que ce soit très informatif.

Il faudrait l'enlever ou loguer truc du style "isSupported = [TRUE/FALSE]" ou "Version not supported" non ?

Reply:
Just a silly debug line I forgot to remove... thanks !